lms/templates/class_dashboard/d3_stacked_bar_graph.js

```diff
@@ -439,6 +439,12 @@ edx_d3CreateStackedBarGraph = function(parameters, svg, divTooltip) {
       .attr("transform","rotate(-90)")
       .style("text-anchor","end")
       .text(gettext("Number of Students"));
+    
+    graph.svgGroup.xAxis
+    .on("mouseover", function(d) {
```

Comment:
Per the existing convention in this file, this block (the `on` call), should be indented another level.